Account.rb

```diff
@@ -1,6 +1,8 @@
 require 'json'
-class Accounts
+require 'Transactions'
 
```

Comment:
Why you require 'Transactions' in Account.rb

Reply:
I thought it is similar with Java files.